docs/source/plugins.rst

```diff
@@ -218,10 +218,10 @@ plugin:
      - `DummyEvaluator <https://github.com/mlflow/mlflow/blob/branch-1.23/tests/resources/mlflow-test-plugin/mlflow_test_plugin/dummy_evaluator.py>`_.
    * - [Experimental] Plugins for custom mlflow server flask app configuration `mlflow.server.app <https://github.com/mlflow/mlflow/blob/v2.2.0/mlflow/server/__init__.py#L30>`_.
      - mlflow.app
-     - The entry point name (e.g. ``mlflow_test_plugin:app``) specifies a customized flask application. This can be useful for implementing
+     - The entry point name (e.g. ``custom_app=mlflow_test_plugin.app:app``) specifies a customized flask application. This can be useful for implementing
```

Comment:
We can describe it more clear: in the example ``custom_app=mlflow_test_plugin.app:app``, the `custom_app` is the --app-name value.

Reply:
Updated